src/components/data-management/Header.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Row, Typography, Space, Button, Dropdown, Col,
+} from 'antd';
+import {
+  updateProject,
+} from '../../redux/actions/projects'; import DownloadData from './DownloadData';
+
+const { Title, Text, Paragraph } = Typography;
+
+const Header = (props) => {
+  const {
+    activeProjectUuid, createMetadataColumn, isAddingMetadata, canLaunchAnalysis,
+    setUploadModalVisible, openAnalysisModal,
+  } = props;
+  const dispatch = useDispatch();
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]) || false;
+  const projects = useSelector((state) => state.projects);
+  const changeDescription = (description) => {
+    dispatch(updateProject(activeProjectUuid, { description }));
+  };
+  return (
+    <>
+      <Row style={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Title level={3}>{activeProject.name}</Title>
+        <Space>
+          <Button
+            disabled={projects.ids.length === 0}
+            onClick={() => setUploadModalVisible(true)}
+          >
+            Add samples
+          </Button>
+          <Button
+            disabled={
+              projects.ids.length === 0
+            || activeProject?.samples?.length === 0
+            || isAddingMetadata
```

Comment:
this `isAddingMetadata` doesn't read very well, or at least it is not obvious to me what it is about. Can you explain?

Reply:
We use it when the user is in the middle of adding metadata - disable the Add metadata button so they can't click it again while creating metadata